electives/18623.md

```diff
@@ -0,0 +1,66 @@
+
+The TAs will hold lab sessions on Fridays, some of which are dedicated to helping you out with the homework. Be sure to attend those, and come with lots of questions! That is a great time to get some clarification for the homework, as well as lecture concepts in general.
+
+Ultimately, the homeworks have a lower weight on your grade, relative to the projects and exams. However, you should still give a good effort on them - be sure to go back and learn what you did not understand fully when doing the homework. This will help you when it comes time to tackle the projects and exams.
+
+## Exams
+
+There are two exams in the course. The first exam covers roughly the first ten lectures, and the second exam covers roughly the next ten lectures (though the second exam depends upon material from the first).
+
+The exams are very reflective of what is covered in the lectures and on the homeworks. As you go through each lecture, be sure to take notes on the different examples covered. Additionally, be sure to spend time reviewing the homeworks after you have completed them. Both will help prepare you for what is covered on the exams.
+
+The exams are similar in weight as the projects for your grade, so it is advised to prepare well for them.
+
+## Projects
+
+The projects are the "meat" of this course. Through these projects, you will learn how to use Cadence for circuit simulation and layout, as well as how to approach an analog IC design task.
+Consequently, you will likely spend a significant amount of time on the projects - but hopefully, you will be learning through doing. You will be asked to meet a number of design specifications for both projects; the challenge comes in meeting all specifications, since there are often key tradeoffs in design.
+
+The projects are done in pairs of two. Since you are working in Cadence, it can be tricky to divide responsibilities, but you should still be able to break the projects into subcomponents that you can both work on and then integrate.
+Additionally, both projects will have a design review, which will allow you to get feedback on your approach before you begin your implementation. Use this time to get as much advice as possible!
+
+You will have a few short lab sessions that will catch you up on the basics of Cadence simulation and layout. But you'll really start to become familiar with its tools through working on the projects.
+Note that the projects are done over roughly the course of one month each. This may sound like a lot of time, but you will also have homeworks and exams in the course during this time - the earlier you start on the projects, the better!
+
```

Comment:
Is it also worth mentioning about how to access Cadence? Using lab computers is the best way to do it, and it's so much faster than using a VM. I used VMs because I was often on the road (interviewing, traveling), and it was a PAIN to do anything. Might be worth mentioning that this is not a very remote-friendly class (only if the VM Cadence experience is still bad)

Reply:
They actually have it so that we can remote access into the ECE clusters, and run the Cadence software remotely from our computer! That said, it can still be a bit laggy, especially on slower computers. I added a section suggesting that folks try setting up the remote access early, or just use the physical computers in the labs